backend/routes/notice.js

```diff
@@ -23,19 +24,55 @@ router.post("/", auth, function (req, res, next) {
     })
     .catch((err) => {
       res.status(500).json({
-				message: 'Something went wrong'
+        message: "Something went wrong",
       });
       console.log(err);
     });
 });
 
-// Returns all the Notices (sorted : latest first)
+// Returns 10 the Notices (sorted : latest first)
 router.get("/", (req, res, next) => {
+  Notice.find({})
+    .sort({ createdAt: -1 })
+    .limit(10)
+    .then((result) => {
+      res.status(200).json({
+        result: result,
+        message: 'OK'
+      });
+    })
+    .catch((err) => {
+      console.log(err);
+      res.status(500).json({
+        message: "Something went wrong",
+      });
+    });
+});
+
+// Returns all the notices
+router.get("/all", (req, res, next) => {
```

Comment:
I wonder where we'll be using this.. :thinking:

Reply:
I'm also not sure of this, we'll be allowing all previous notices or not!

Reply:
No harm in having an option to retrieve all of em..